src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/PackAllRids.targets

```diff
@@ -2,6 +2,8 @@
   <!-- We are defining our own Pack target, so opt out of the SDK version. -->
   <PropertyGroup>
     <ImportNuGetBuildTasksPackTargetsFromSdk>false</ImportNuGetBuildTasksPackTargetsFromSdk>
+    <_RoslynPublishReadyToRun>false</_RoslynPublishReadyToRun>
+    <_RoslynPublishReadyToRun Condition="'$(Configuration)' == 'Release'">true</_RoslynPublishReadyToRun>
```

Comment:
I don't know if you want to extract out this condition back to the main .csproj and use it in line 47 there, so that way we don't have this condition expressed twice. This .targets file is included in the csproj at the end so you'd be able to use it in the MSBuild invocation later.